articles/mfa/guides/configure-email-universal-login.md

```diff
@@ -19,17 +20,17 @@ Users do not need to explicitly enroll with email MFA. They will get be able to
 
 Note that Email is not true <dfn data-key="multifactor-authentication">multi-factor authentication (MFA)</dfn> as it does not represent a different factor than the password. It does not represent 'something I have' or 'something I am', but rather just another 'something I know' (the email password). It is also weaker than other factors, in that it's only as secure as the email itself (e.g. is it encrypted end-to-end?).
 
-## End-user experience
+#### End-user experience
```

Comment:
Why is this and the ones below changed to h4? the only thing above it is the h1, they should all be h2